arcgis-ios-sdk-samples/Maps/Display device location with NMEA data sources/README.md

```diff
@@ -30,10 +34,23 @@ Tap "Start" to parse the NMEA sentences into a simulated location data source, a
 
 ## About the data
 
-A list of NMEA sentences is used to initialize a `SimulatedNMEADataSource` object. This simulated data source provides NMEA data periodically, and allows the sample to be used on devices without a GPS dongle that produces NMEA data.
+A list of NMEA sentences is used to initialize a `SimulatedNMEADataSource` object. This simulated data source provides NMEA data periodically, and allows the sample to be used on devices without a GNSS receiver that produces NMEA data.
 
 The route taken in this sample features a [2-minute driving trip around Redlands, CA](https://arcgis.com/home/item.html?id=d5bad9f4fee9483791e405880fb466da).
 
+## Additional information
+
+To support GNSS device connection in an app, here are a few steps:
+
+* Enable Bluetooth connection in the Settings, or connect via Lightning connector.
```

Comment:
```suggestion
* Enable Bluetooth connection in the device settings or connect via Lightning connector.
```
Is Lightning the correct term here? I think some iOS devices only support Thunderbolt.

Reply:
Hmm good point. I'll change it to cable.